rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -658,68 +766,140 @@ def _use_default_label_features(self, label_ids: np.ndarray) -> List[FeatureArra
 
     def _create_model_data(
```

Comment:
Same, this has now become a very large function. Please break it down into smaller functions.